src/main/java/io/quarkus/search/app/util/GitCloneDirectory.java

```diff
@@ -152,12 +235,6 @@ private void update(Git git, String remoteName) {
                         .setProgressMonitor(LoggerProgressMonitor.create(log,
                                 "Fetching into '" + directory + "' (" + branches.pages() + "): "))
                         .call();
-
-                // pull the sources branch, to update the working directory
-                git.pull()
-                        .setProgressMonitor(LoggerProgressMonitor.create(log,
-                                "Pulling into '" + directory + "' (" + branches.sources() + "): "))
-                        .call();
```

Comment:
Nice. But shouldn't you remove the checkout too? We no longer need to change the working directory, do we?

Reply:
good point 👍🏻 😃  thanks!